CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to Docker open source projects
+
+Want to hack on this project? Awesome! Here are instructions to get you started.
+
+This project is a part of the [Docker](https://www.docker.com) project, and follows
```

Comment:
Looks like containerd. https://github.com/docker/containerd/blob/master/CONTRIBUTING.md#successful-changes

Let me know and I'll submit a PR.